lib/gerry/client/access.rb

```diff
@@ -0,0 +1,17 @@
+require 'erb'
+
+module Gerry
+  class Client
+    module Access
+      # Get access rights for the specified project
+      #
+      # @param [VarArgs] projects the project names
+      # @return [Hash] the list of access rights
+      def access(*projects)
+        projects = projects.flatten.map { |name| ERB::Util.url_encode(name) }
```

Comment:
Using ERB::Util over similar functionality in URI here is a bit of a surprise. Not necessarily bad, just odd given that we're not really using ERB for templating purposes already.